### PR TITLE
Update Kafka tutorial to work with Apache Kafka

### PR DESCRIPTION
**Pull request by dalelane**
Minor update to the Kafka tutorial to remove the dependency on
Confluent's distribution of Kafka, and instead work with vanilla
Apache Kafka.

Signed-off-by: Dale Lane <dale.lane@uk.ibm.com>

### CODE REVIEW

**Comment by dalelane** [comment]
@googlebot I signed it!

**Comment by dalelane** [comment]
context for this change: https://github.com/tensorflow/io/issues/802#issuecomment-755230465

**Comment by dalelane** [comment]
@kvignesh1420 As discussed - a small update to the Kafka tutorial that gets it to run using vanilla Apache Kafka

File: docs/tutorials/kafka.ipynb [hunk 1]
```diff
@@ -104,7 +106,9 @@
       "outputs": [],
       "source": [
         "!pip install tensorflow-io\n",
-        "!pip install kafka-python"
+        "!pip install kafka-python\n",
+        "!pip install sklearn\n",
+        "!pip install pandas"
```
Comment (kvignesh1420) [inline docs/tutorials/kafka.ipynb:111]: @dalelane `sklearn` and `pandas` are pre-installed in colab. You can remove these lines.

**Review by kvignesh1420 — CHANGES_REQUESTED** [review]
Please check comments.

**Comment by dalelane** [comment]
updated - thanks for the review!

**Review by kvignesh1420 — APPROVED** [review]
LGTM. I tried running the updated notebook and it ran without any errors.

**Pull request merged**